OloEngine/src/OloEngine/Renderer/Commands/CommandMemoryManager.cpp

```
@@ -207,4 +237,136 @@ namespace OloEngine
 
         return allocator;
     }
+
+    // ========================================================================
+    // Per-Worker Allocator API Implementation
+    // ========================================================================
+
+    CommandAllocator* CommandMemoryManager::GetWorkerAllocator(u32 workerIndex)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)